gcp/appengine/blog/content/posts/supporting-debian-security-tracker-data/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Supporting Debian Security Tracker data"
+date: 2024-06-18T00:00:00Z
+draft: false
+author: Holly Gong
+---
+
+OSV.dev aims to be a comprehensive database for all known vulnerabilities in open source ecosystems. One of the more recent areas of focus is Linux distributions, which are important in the context of container scanning. To enable comprehensive scanning support for Debian based containers, we've integrated more vulnerability data from Debian Security Tracker into our OSV database.
+
+![Screenshot of Debian Security Tracker data on OSV.dev](debian.png "Screenshot of Debian Security Tracker data on OSV.dev")
+A [sample CVE entry](https://osv.dev/vulnerability/CVE-2024-2466) with Debian information. 
+
+
+## What is Debian Security Tracker?
+The [Debian Security Tracker](https://security-tracker.debian.org/tracker/) is a database maintained by Debian's security team to track Debian related security vulnerabilities. It aggregates information from various sources, including [Debian Security Advisories](https://www.debian.org/security/#DSAS) (DSAs), the [Common Vulnerabilities and Exposures](https://www.cve.org/) (CVE) database, the [National Vulnerability Database](https://nvd.nist.gov/) (NVD), and the [Debian bug tracking system](https://www.debian.org/Bugs/) (BTS). This makes the security tracker comprehensive, providing a complete record of reported Debian security vulnerabilities.
+
+## Why do we need it?
+Previously, OSV relied solely on Debian Security Advisories (DSAs) and Debian Long Term Support Advisories (DLAs) for Debian related vulnerability information. While these are official sources, they only cover issues that have been formally [acknowledged and patched](https://www.debian.org/doc/manuals/securing-debian-manual/dsa.en.html) by the Debian team. This left a blind spot in our scanning reports – unfixed vulnerabilities, where the security issue has been reported but a fix has not been made available yet in Debian.
```

Comment:
A little bit of a nit, Debian's a project, the security team largely handle security fixes, so either refer to it overall as Debian, or the Debian security team in this instance. Also differentiate between OSV and OSV.dev. Also link to more information about the current conversion process, for the curious.

```suggestion
Previously, OSV.dev [relied](https://google.github.io/osv.dev/data/#converted-data) solely on Debian Security Advisories (DSAs) and Debian Long Term Support Advisories (DLAs) for Debian related vulnerability information. While these are official sources, they only cover issues that have been formally [acknowledged and patched](https://www.debian.org/doc/manuals/securing-debian-manual/dsa.en.html) by Debian. This left a blind spot in our data coverage – unfixed vulnerabilities, where the security issue has been reported but a fix has not been made available yet in Debian.
```